libs/community/langchain_community/document_loaders/mongodb.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         connection_string: str,
         db_name: str,
         collection_name: str,
+        field_names: Optional[List[str]] = None,
```

Comment:
Given that this is a named argument could you put it after the `*` and change the type to Optional[Sequence[str]]

Reply:
Good point!

Reply:
Fixed